Paco-Server/src/com/google/sampling/experiential/server/EventRetriever.java

```diff
@@ -80,6 +82,7 @@
   private static EventRetriever instance;
   private static final Logger log = Logger.getLogger(EventRetriever.class.getName());
   private static CloudSQLDao cloudSqlDaoImpl = new CloudSQLDaoImpl();
+  private static DateTimeFormatter df = org.joda.time.format.DateTimeFormat.forPattern(TimeUtil.DATETIME_FORMAT).withOffsetParsed();
```

Comment:
The package for DateTimeFormat is imported already so you can remove the full package name on the class.